client/packages/common/src/intl/locales/en/common.json

```
@@ -549,6 +549,8 @@
   "items": "Items",
   "label.abbreviation": "Abbreviation",
   "label.acknowledged": "Acknowledged",
+  "label.percent-discount": "percent discount",
```

```suggestion
  "label.percent-discount": "Insurance discount percentage",
```

Until we re-do it with the sub table structure with an Insurance Section